examples/demo.js

```diff
@@ -1,39 +1,48 @@
 import React, { Component } from 'react'
 import clsx from 'clsx'
 import Villain from '@/index.js'
-import {version} from '../package.json';
+import { version } from '../package.json'
 import './demo.css'
+import ToggleThemeButton from './ToggleThemeButton'
 
 class Demo extends Component {
-
   state = {
     file: '/build/testFile/Example-archive.zip',
     options: {
       preview: 0,
-      theme: {type: 'select', value: 'Light', options: ['Dark', 'Light']},
+      theme: { type: 'select', value: 'Light', options: ['Dark', 'Light'] },
       mangaMode: false,
       allowFullScreen: true,
       autoHideControls: false,
       allowGlobalShortcuts: false,
-    }
+    },
+    theme: 'light',
   }
 
   constructor(props) {
     super(props)
   }
 
+  toggleTheme = () => {
+    const theme = this.state.theme === 'light' ? 'dark' : 'light'
+    document.documentElement.classList.add('color-theme-in-transition')
+    this.setState({ theme })
+    document.documentElement.setAttribute('data-theme', theme)
+    window.setTimeout(() => {
+      document.documentElement.classList.remove('color-theme-in-transition')
+    }, 300)
```

Comment:
Why a delay on this ?

Reply:
I added a delay for a smooth transition to the dark mode. You can run it and see. I thought this would look good thats why. I'd remove it if you want :)

Reply:
I don't see the need for animation / transition, anyway you should update state inside the timeout function, and let react handle the rest with a conditional rendering.  
This example is just for testing development version.